src/presets/firebase/preset.ts

```diff
@@ -88,7 +88,7 @@ const firebaseAppHosting = defineNitroPreset(
           stringifyYAML({
             version: "v1",
             runConfig: {
-              runCommand: "node .output/server/index.mjs",
+              runCommand: `node ${join(nitro.options.output.serverDir, "index.mjs")}`,
```

Comment:
Let's generate it relative to `options.rootDir` (or `options.workspaceDir` depending on current app hosting behavior) instead of absolute path.